infra/charts/feast-feature-server/templates/otel-collector.yml

```diff
@@ -0,0 +1,56 @@
+# API reference https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md
+# Refs for v1beta1 config: https://github.com/open-telemetry/opentelemetry-operator/issues/3011#issuecomment-2154118998
+{{- if .Values.metrics }}
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
```

Comment:
IMO, this chart should not be handling the installation of the collector. instead, we could offer the user a set collector related values under the `metrics` umbrella which would allow them to point to a collector if they so choose.